src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +426,175 @@ public void SetFocus(bool focused)
             Interop.ChromiumEwk.ewk_view_focus_set(_realHandle, focused);
         }
 
+        /// <summary>
+        /// Gets size of the content.
+        /// </summary>
+        /// <returns> size of the coordinate.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public Size ContentsSize
+        {
+            get
+            {
+                int width, height;
+                Interop.ChromiumEwk.ewk_view_contents_size_get(_realHandle, out width, out height);
+                return new Size(width, height);
+            }
+        }
+
+        /// <summary>
+        /// Get cookies for URL.
+        /// </summary>
+        /// <param name="url">The URL.</param>
+        /// <returns>'cookie string' on success, otherwise 'null'.</returns>
```

Comment:
Could you show some example of result?
Usually cookie value has key and value pair
How to express key and value in returning string?

And How about returning dictionary instead of string?

Reply:
According to [RFC 6265](https://tools.ietf.org/html/rfc6265#section-4.1.1), cookie string is defined by the following grammar:
cookie-string = cookie-pair *( ";" SP cookie-pair )
It is well-known grammar, so the API user can convert from string to dictionary easily.
I think that this API is enough, and Android provide similar API, `public abstract String getCookie (String url)`, as well.

Reply:
Ok, I understand.

and How to set cookies?

Reply:
And why it is provided on WebView instead of CookieManager or Context?

Reply:
It originally owned by Context(CookieManager)
``` c++
char* ewk_view_get_cookies_for_url(Evas_Object* view, const char* url)
{
  EWK_VIEW_IMPL_GET_OR_RETURN(view, impl, 0);
  EINA_SAFETY_ON_NULL_RETURN_VAL(url, 0);
  std::string cookiesForURL;
  cookiesForURL = impl->context()->cookieManager()->GetCookiesForURL(std::string(url));
  if (cookiesForURL.empty())
    return NULL;
  return strndup(cookiesForURL.c_str(), cookiesForURL.length());
}
```

Reply:
Good point, I agree that `GetCookies` is provided on `CookieManager`.
We will accept it. @BasavarajPS , please move `GetCookies` to `CookieManager`, and then create new PR.

And, setting cookie is not supported yet on chromium-efl.

Reply:
Done, I removed GetCookie() from view